audit-service/README.md

```diff
@@ -0,0 +1,108 @@
+<!---
+Copyright 2020 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# <img src="../logos/logo.svg" width="180">
+
+# Audit Service
+
+The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages
+contain all the details of the initial request and any other relevant information. This information
+will be passed to any local audit services that have been implemented. This service does not have any
+output Kafka topics.
+
+## Message model
+
+| AuditSuccessMessage | AuditErrorMessage | 
```

Comment:
Can you explain how we can tie both the success and error messages back to the original request?

Reply:
Added information about the `token`.